src/components/Homepage.js

```diff
@@ -0,0 +1,16 @@
+import '../css/home-page.css';
+import { HomePageCategories, HomePageCards, HomePageHero } from '../components/Homepage/';
+
+
+const HomePage = () => {
+
```

Comment:
make a folder directly under src and name it pages, in that folder make your pages , like this file.

Reply:
will result in a better file structuring

Reply:
Yes, will do that, easier to access and read.